po/vi.po

```diff
@@ -3584,168 +3584,172 @@ msgid ""
 "The [Collatz Sequence](https://en.wikipedia.org/wiki/Collatz_conjecture) is "
 "defined as follows, for an arbitrary n"
 msgstr ""
+"[Chuỗi Collatz](https://vi.wikipedia.org/wiki/"
+"Gi%E1%BA%A3_thuy%E1%BA%BFt_Collatz) được định nghĩa như sau: với một giá trị "
```

Comment:
I gave the wiki link in Vietnamese instead, is that ok?

Reply:
I think that's great! This flexibility is exactly why I haven't gone to the trouble of removing the links from the translation data.